report/Motivation/Motivation.tex

```diff
@@ -25,11 +25,18 @@ \section{Speculations on Agile's Impact}
 
 \textbf{Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.}
 This principle indicates that the developers should deliver vertical slices of the software to the customer as soon as possible and continuously.
-We assume that this will allow the developers to validate the software early and learn whether the software is correct in terms of actual usage instead of in terms of specified requirements.
+A vertical slice of software refers to a functionality that is implemented across all the layers of the software - let us say a software consist of three layers: Data Management layer, Business logic layer, and User Interface layer. 
+Then a vertical slice of the authentication component could be the login part. Which would require a User interface for performing the login activity, the business logic for actually authenticating a user, and the data management for storing the users' credentials. It would not necessarily require a logout functionality.
```

Comment:
```suggestion
Then a vertical slice of the authentication component could be the login part. 
This require a user interface for performing the login activity, the business logic for actually authenticating a user, and the data management for storing the users' credentials. 
It would not necessarily require a logout functionality.
```